src/absil/il.fsi

```diff
@@ -220,6 +222,11 @@ type ILTypeSpec =
     /// The name of the type in the assembly using the '.' notation for nested types.
     member FullName: string
     
+    /// The basic qualified name of the type in the assembly.
+    member internal BasicQualifiedName: string
```

Comment:
I think we don't need to expose this anymore.

Reply:
ooo good catch